packages/react-components/src/components/VideoGallery/styles/VideoGalleryResponsiveHorizontalGallery.styles.ts

```diff
@@ -4,16 +4,7 @@
 import { IStyle } from '@fluentui/react';
 import { _pxToRem } from '@internal/acs-ui-common';
 import { HorizontalGalleryStyles } from '../../HorizontalGallery';
-
-/**
- * Small floating modal width and height in rem for small screen
- */
-export const SMALL_FLOATING_MODAL_SIZE_PX = { width: 64, height: 88 };
-
-/**
- * Large floating modal width and height in rem for large screen
- */
-export const LARGE_FLOATING_MODAL_SIZE_PX = { width: 160, height: 120 };
```

Comment:
Fixing my bug on your branch. Lesson is never have the same variable defined twice :)
https://user-images.githubusercontent.com/79475487/207985251-3e3fb24d-0034-4f1c-819d-6fc338a219ea.mp4
Thank goodness for chromatic!

Reply:
Hahaha, Nice!! :)